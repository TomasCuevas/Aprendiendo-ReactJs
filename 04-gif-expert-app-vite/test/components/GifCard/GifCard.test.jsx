import { render } from "@testing-library/react";
import { GifCard } from "../../../src/components/GifCard/GifCard";

describe('Pruebas en <GifCard />', () => {

  const title = 'Titulo del GIF';
  const url = 'https://urldelgif.com';
  
  test('hacer snapshot del componente', () => {
    const { container } = render(<GifCard title={title} url={url} />);
    expect(container).toMatchSnapshot();
  });
  
})