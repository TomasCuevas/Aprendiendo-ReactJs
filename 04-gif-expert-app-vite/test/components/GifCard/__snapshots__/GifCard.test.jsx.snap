// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifCard /> hacer snapshot del componente 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Titulo del GIF"
      src="https://urldelgif.com"
    />
    <p>
      Titulo del GIF
    </p>
  </div>
</div>
`;
